extensions/amp-story/amp-story-analytics.md

```diff
@@ -56,6 +56,81 @@ Consult your vendor's documentation for more specific details on how to set this
 
 The `story-last-page-visible` trigger is fired when the last page in the story is shown to the user. This can be used to measure completion rate.
 
+### Tooltip focus trigger (`"on": "story-focus"`)
+
+The `story-focus` trigger is fired when clicking an element that opens a tooltip.
+
+The current elements trackable by the `story-focus` and `story-click-through` triggers are:
+
+- `<a>`
+- `<amp-twitter>`
+
+To use it, specify the trigger on your `"triggers"` property of your analytics configuration, accompanied by the `"tagName"` of the element you want to track.
+
+Example:
+
+```
+<amp-analytics id="my-analytics">
+  <script type="application/json">
+    {
+      "requests": {
+        "click": "https://example.com/my-endpoint"
+      },
+      "triggers": {
+        "trackFocusState": {
+          "on": "story-focus",
+          "tagName": "a",
+          "request": "click"
+        }
+      }
+    }
+  </script>
+</amp-analytics>
+
+```
+
+### Tooltip click through trigger (`"on": "story-click-through"`)
+
+The `story-click-through` trigger is fired when clicking on a tooltip.
+
+To use it, specify the trigger on your `"triggers"` property of your analytics configuration, accompanied by the `"tagName"` of the element you want to track.
+
+Example:
+
+```
+<amp-analytics id="my-analytics">
+  <script type="application/json">
+    {
+      "requests": {
+        "click": "https://example.com/my-endpoint"
+      },
+      "triggers": {
+        "trackClickThrough": {
+          "on": "story-click-through",
+          "tagName": "a",
+          "request": "click"
+        }
+      }
+    }
+  </script>
+</amp-analytics>
+
+```
+
+### Story open trigger (`"on": "story-open"`)
+
+The `story-open` trigger is fired when opening a drawer or dialog inside a story. The components that are currently trackable by this are:
```

Comment:
Hey ,
Is there any way to distinguish which dialog box opens, i.e either info dialog or share dialog.If that's is possible Please do revert will be helpful

Reply:
Thanks Enrique, that will be helpful
Ya ya no issues I will request the feature specifying the same & will notify you there response.